ext/mysqli/tests/mysqli_get_client_stats.phpt

```diff
@@ -4,7 +4,33 @@ mysqli_get_client_stats()
 mysqli
 --SKIPIF--
 <?PHP
-require_once 'skipifconnectfailure.inc';
+    require_once __DIR__ . '/test_setup/test_helpers.inc';
+    $link = mysqli_connect_or_skip();
+    // We run some operations that need CREATE SERVER and CREATE DATABASE in
+    // this test, so check if we can do that. We don't check if we can merely
+    // create a table; if that doesn't work with the provided database, many
+    // more tests would fail.
+
+    // It seems DML can be privilege checked on prepare, but DML can't be, so
+    // we need to execute and clean up.
```

Comment:
I don't really understand this comment.

Reply:
IIRC, you should be able to do something like `mysqli_prepare("INSERT INTO TABLE")`, and it does permission checks at that stage, but seemingly not for `CREATE DATABASE` when I tested it.